pkg/ast/spl/spl.peg

```diff
@@ -3255,16 +3379,25 @@ FieldNameList <- first:FieldName rest:(COMMA FieldName)* {
 
 TimeModifiers <- EarliestAndLatest / EarliestOnly
 
-EarliestAndLatest <- CMD_EARLIEST EQUAL earliestTime:AbsoluteTimestamp SPACE CMD_LATEST EQUAL latestTime:AbsoluteTimestamp !. {
-    startEpoch, err := ConvertCustomDateTimeFormatToEpochMs(earliestTime.(string))
+EarliestAndLatest <- CMD_EARLIEST EQUAL earliestTime:(AbsoluteTimestamp / RelativeTimestamp) SPACE CMD_LATEST EQUAL latestTime:(AbsoluteTimestamp / RelativeTimestamp) !. {
+     var startEpoch int64 = 0
+     var endEpoch int64 = 0
+    var err error
```

Comment:
nit: something's wrong with the indentation. Maybe you're using a combination of tabs and spaces? Unlike the .go files, for this .peg file we use spaces to indent

Reply:
addressed in the latest commit